makefile

```diff
@@ -13,10 +13,30 @@ omim.sssom.tsv: omim.json
 	sssom parse omim.json -I obographs-json -m data/metadata.sssom.yml -o omim.sssom.tsv
 	make cleanup
 
+mondo_exactmatch_omim.sssom.tsv:
+	wget "http://purl.obolibrary.org/obo/mondo/mappings/mondo_exactmatch_omim.sssom.tsv" -O $@
+
+mondo_exactmatch_omimps.sssom.tsv:
+	wget "http://purl.obolibrary.org/obo/mondo/mappings/mondo_exactmatch_omimps.sssom.tsv" -O $@
+
+%.sssom.owl: %.sssom.tsv
+	sssom convert $< -O owl -o $@
+
 # More commands / goals  -----------------------------------------------------------------------------------------------
 # Create mapping artefact(s)
-omim.json:
-	robot convert -i omim.ttl -o omim.json
+omim.json: omim.owl
+	robot convert -i $< -o omim.json
+
+# Create OWL artefact, but adding HGNC links alongside the OMIM genes and
+# Mondo mappigs alongside the OMIM diseases
+omim.owl: omim.ttl mondo_exactmatch_omim.sssom.owl mondo_exactmatch_omimps.sssom.owl
```

Comment:
## `omim.owl`
### Questions
1. Why `.owl` and not `.ttl` for this serialization format?
Is it simply because that's more standard / easier to output from `robot`?

2. Why do we want two different ontologization outputs?
I.e. `omim.owl` and `omim.ttl`. I see that `omim.owl` has _ HGNC links alongside the OMIM genes and
# Mondo mappings alongside the OMIM diseases_. But why not just do only `omim.owl` in the release?

Reply:
No reason, I kinda never want to yank a release file if it already exists, but if you think its better we can stop attaching the file to the release (the `Makefile` wont change though, as omim.owl depends on omim.ttl).

Reply:
MAde the change above.